CHANGES.txt

```diff
@@ -1,4 +1,5 @@
 Current
+New: Upgrade to gradle5
```

Comment:
Can you please include a reference to https://github.com/cbeust/testng/issues/1968 which talks about this upgrade ?

Reply:
@ajayk - Fair enough.